docs/man/docker-run.1.md

```diff
@@ -197,6 +199,12 @@ ENTRYPOINT.
                                'container:<name|id>': reuses another container shared memory, semaphores and message queues
                                'host': use the host shared memory,semaphores and message queues inside the container.  Note: the host mode gives the container full access to local shared memory and is therefore considered insecure.
 
+**-l**, **--label**=[]
+   Set meta data on the container (e.g., --label=com.example.key=value)
```

Comment:
Did you me to jam this all together `label=com.example.key=value`?

--->try

Set container metadata, for example, `label=com.example`